docs/release-notes/8.7.asciidoc

```diff
@@ -10,6 +10,14 @@
 ==== Known issues
 * After alerts are generated for the first time, you may have to refresh your browser before your alert data appears on pages that use data views (for example, Timeline). Navigating between pages will not work (https://github.com/elastic/security-docs/issues/3046[#3046]).
 
+
+*{agent} upgrade from 8.6.2 to 8.7.2 can fail:*
+When upgrading {agents} running the Osquery Manager integration from 8.6.2 to 8.7.0, they can get stuck in the `Updating` state (https://github.com/elastic/elastic-agent/issues/2433[#2433]). To prevent this, delete the Osquery Manager integration from your {agents} before upgrading them to 8.7.0, then re-add it after you finish upgrading.
```

Comment:
This might be too verbose, but I wanted to clarify that the user would be going through the agent upgrade process, not the Osquery integration upgrade.  

```suggestion
* The {agent} upgrade from 8.6.2 to 8.7.0 might fail for {agents} running the Osquery Manager integration ({agents} can get stuck in the `Updating` state) (https://github.com/elastic/elastic-agent/issues/2433[#2433]). To prevent this, delete the Osquery Manager integration from your {agents} before upgrading them to 8.7.0, then re-add it after you finish upgrading.
```